scram-tools.file/tools/TOPO/topo.xml

```diff
@@ -0,0 +1,6 @@
+<tool name="TOPO" version="@TOOL_VERSION@" revision="1">
+  <client>
+    <environment name="TOPO_BASE" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR"       default="$TOPO_BASE/lib64"/>
```

Comment:
```suggestion
<tool name="@TOOL@" version="@TOOL_VERSION@" revision="1">
  <client>
    <environment name="@TOOL_BASE@" default="@TOOL_ROOT@"/>
    <environment name="LIBDIR"       default="$@TOOL_BASE@/lib64"/>
```

please use these generic keywords which packaging system can automatically repalce with the right values